helm_deploy/values-preprod.yaml

```diff
@@ -1,37 +1,26 @@
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+---
+# Environment specific values, override helm_deploy/hmpps-manage-users/values.yaml
 
-replicaCount: 4
+generic-service:
+  replicaCount: 4
```

Comment:
default for replicas is 4 in the generic service I think

Reply:
Looks like it's 2 in values.yaml (maybe we should change it to 4?)